node/src/cli.rs

```diff
@@ -82,6 +83,24 @@ pub struct ProviderConfigurations {
     )]
     pub provider_type: Option<ProviderType>,
 
+    /// Maximum storage capacity of the provider (bytes).
+    ///
+    /// Default: 4294967295 bytes (4GiB)
+    #[clap(long, required_if_eq_any([
+        ("provider_type", "msp"),
+        ("provider_type", "bsp")
+    ]))]
+    pub max_storage_capacity: Option<StorageDataUnit>,
+
+    /// Jump capacity (bytes).
+    ///
+    /// Default: 4294967295 bytes (4GiB)
+    #[clap(long, required_if_eq_any([
+        ("provider_type", "msp"),
+        ("provider_type", "bsp")
+    ]))]
+    pub jump_capacity: Option<StorageDataUnit>,
```

Comment:
We probably want to make these NOT optional and with default types. We can have our testing setup give them default values so that it is not cumbersome for us, but BSP operators should be aware of these configurations.

Reply:
We have concluded the opposite from one of our meetings where we want to have it be required if you run the node as a storage provider and force node operators to be aware of these configurations to avoid any misconfigured storage capacities. 

It is only considered optional if the node is run as a user as of right now. It will fail and require you to provide these options otherwise. Also it didn't take me long to add these options to the CI. It is more clear as well IMO that we have this explicitely set in the CI.

Reply:
I'll update the comment to reflect.

Reply:
Sorry, my comment was not clear. I meant what you said, in case the provider types is "msp" or "bsp", then they should be required. I thought that was not the case here in the code, but it is. So great 👌🏼